qt-ifw/config/config.xml

```diff
@@ -4,9 +4,15 @@
     <Version>@VERSION@</Version>
     <Title>MSYS2 @BITNESS@bit</Title>
     <Publisher>The MSYS2 Developers</Publisher>
+    <ControlScript>control.js</ControlScript>
     <StartMenuDir>MSYS2 @BITNESS@bit</StartMenuDir>
     <RunProgram>@TargetDir@/msys2_shell.cmd</RunProgram>
     <RunProgramArguments></RunProgramArguments>
     <InstallerApplicationIcon>../../msys2</InstallerApplicationIcon>
     <InstallerWindowIcon>../../msys2</InstallerWindowIcon>
+    <InstallActionColumnVisible>false</InstallActionColumnVisible>
+    <MaintenanceToolName>Uninstall_MSYS2</MaintenanceToolName>
```

Comment:
hm, what about just a plain "uninstall"

Reply:
If you prefer, sure, that works. What it uninstalls is certainly implied by its location on disk.

Reply:
yes please